src/components/BaseModal.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Modal from 'react-modal';
+import './BaseModal.css';
+import { PageSubHeading } from './PageSubHeading';
+import CloseIconComponent from './CloseIconComponent';
+
+const customStyles = {
+  overlay: {
+    position: 'fixed',
+    top: '0',
+    left: '0',
+    right: '0',
+    bottom: '0',
+    backgroundColor: 'rgba(0, 0, 0, 0.5)',
+    overflowY: 'auto',
+  },
+
+  content: {
+    position: 'relative',
+    top: '20%',
+    maxHeight: '60%',
+    margin: '0 auto',
+    maxWidth: '60%',
+    border: '1px solid rgb(204, 204, 204)',
+    background: 'rgb(255, 255, 255)',
+    overflow: 'auto',
+    borderRadius: '4px',
+    outline: 'none',
+    padding: '10px 20px 20px 20px',
+  }
+};
+
+Modal.setAppElement('#root');
+
+export const BaseModal = (props) => {
+  const { disableOkBtn = false } = props;
+  return (
+    <div>
+      <Modal
+        isOpen={props.showModal}
+        onAfterOpen={props.afterOpen}
+        onRequestClose={props.onRequestClose}
+        style={customStyles}
+        contentLabel='Modal'
+      >
+        <div className='modal-header'>
+          <CloseIconComponent
+            closeFn={props.onRequestClose}
+          />
+          <PageSubHeading
+            id={props.id}
+            imgSrc={props.imgSrc}
+            color={props.color}
+            iconSize={props.iconSize}
+            title={props.title}
+            description={props.description}
+          />
+        </div>
+        <div className='modal-content'>
+          {props.children}
```

Comment:
I'm not sure what's going on, but I noticed that on the TranslatedDulModal on the Dataset Catalog which uses the BaseModal, nothing is getting rendered for the `modal-content`.
Local:
<img width="672" alt="image" src="https://github.com/DataBiosphere/duos-ui/assets/68303030/0d507be4-567a-401f-a2d7-e080863c6930">
Dev:
<img width="606" alt="image" src="https://github.com/DataBiosphere/duos-ui/assets/68303030/3e93d267-6850-468c-96e4-904c3f040da3">

Reply:
@aarohinadkarni  Thanks for catching this. I'll look into it.